modules/ccs-aws-provisioned.adoc

```diff
@@ -46,7 +46,7 @@ Volume requirements for each EC2 instance:
 * Input/output operations per second: 900
 
 [id="aws-policy-elastic-load-balancers_{context}"]
-== Elastic load balancers
+== Elastic load balancing (ELB) load balancers
```

Comment:
`Elastic Load Balancing (ELB) load balancers` is better, please capitalize L and B

Reply:
Well spotted. Updated.